src/components/editor/Header.jsx

```diff
@@ -1,10 +1,8 @@
 // Copyright 2018 Stanford University see Apache2.txt for license
 
 import React, { Component } from 'react'
-import SinopiaLogo from '../styles/editorsinopialogo.png'
-import { Link } from 'react-router-dom'
```

Comment:
this imports aren't used.  I found this out as I worked on configuring eslint to do accessibility testing with jsx-a11y eslint plugin.